src/page-modules/assistant/utils.ts

```diff
@@ -72,13 +78,13 @@ export function createTripQuery(
 }
 
 export function parseTripQuery(query: any): TripQuery | undefined {
-  const requiredNumericFields = ['fromLat', 'fromLon', 'toLat', 'toLon'];
-  requiredNumericFields.forEach((field) => {
-    if (typeof query[field] === 'string')
-      query[field] = parseFloat(query[field]);
-  });
-
-  const optionalNumericFields = ['searchTime'];
+  const optionalNumericFields = [
```

Comment:
Is it possible to connect this with the type somewhat? To simplify when we eg refactor so that this gives an error because names don't match.